recipes-core/util-linux/util-linux_2.%.bbappend

```diff
@@ -1,6 +1,6 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 
-DEPENDS_class-target += "shadow-native pseudo-native niacctbase"
```

Comment:
Would you get an error if you tried to build the recipe with a missing dependency?